src/main/java/cs2103/v15_1j/jimjim/command/ChangeCommand.java

```diff
@@ -67,6 +67,7 @@ public LocalTime getNewEndTime() {
 	public UIFeedback undo(ControllerStates conStates) {
 		TaskEvent temp = conStates.masterList.remove(actual);
 		conStates.masterList.add(taskNum-1, backup);
+        conStates.masterList.sort();
```

Comment:
this line and the previous are a bit redundant. We can just `conStates.masterList.add(backup);` to achieve the same effect.